draft-ietf-quic-transport.md

```diff
@@ -2379,15 +2379,19 @@ terminate the connection immediately.  A CONNECTION_CLOSE frame causes all
 streams to immediately become closed; open streams can be assumed to be
 implicitly reset.
 
-After sending a CONNECTION_CLOSE frame, endpoints immediately enter the closing
-state.  During the closing period, an endpoint that sends a CONNECTION_CLOSE
-frame SHOULD respond to any packet that it receives with another packet
-containing a CONNECTION_CLOSE frame.  To minimize the state that an endpoint
-maintains for a closing connection, endpoints MAY send the exact same packet.
-However, endpoints SHOULD limit the number of packets they generate containing a
-CONNECTION_CLOSE frame.  For instance, an endpoint could progressively increase
-the number of packets that it receives before sending additional packets or
-increase the time between packets.
+After sending a CONNECTION_CLOSE frame, an endpoint immediately enters the
+closing state.  During the closing period, an endpoint that sends a
+CONNECTION_CLOSE frame SHOULD respond to any packet that it receives with
+another packet containing a CONNECTION_CLOSE frame, until it receives a packet
```

Comment:
Why would it receive a packet with a CONNECTION_CLOSE frame? The peer won't send a CONNECTION_CLOSE in response to a CONNECTION_CLOSE, so this condition would only occur by pure coincidence.
I don't think we need any text describing this corner case.

Reply:
I agree that this should not be a SHOULD. But we already have a MAY. The current text states that _an endpoint MAY transition from the closing period to the draining period if it receives a CONNECTION_CLOSE frame_.

I think that the text here should be something along the lines of:
* During the closing period, an endpoint that sends a CONNECTION_CLOSE frame SHOULD respond to a packet that *can be AEAD-decrypted* with another packet containing a CONNECTION_CLOSE frame. That action *MAY* cease when it receives a packet containing a CONNECTION_CLOSE frame.

Reply:
Fixed in abfc00c.